setup.py

```diff
@@ -41,4 +41,27 @@
             "rubicon_experiment = rubicon.intake_rubicon.experiment:ExperimentSource",
         ],
     },
+    classifiers=[
+        # How mature is this project? Common values are
+        #   3 - Alpha
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 4 - Beta',
```

Comment:
@mmccarty - we were debating between Beta, or Production/Stable -- what do you think?

Reply:
I think Beta is fine for now given that we are just now open sourcing and might get some activity that would make things less stable.